ui/de-webapp/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+multipart.max-file-size=10000000
+multipart.max-request-size=10000000
```

Comment:
I don't think this file should be checked-in. This is needed just for local dev.